great_expectations/rule_based_profiler/data_assistant/column_value_missing_data_assistant.py

```diff
@@ -98,50 +98,14 @@ def _build_data_assistant_result(
         )
 
     @staticmethod
-    def _build_column_value_missing_rule() -> Rule:
```

Comment:
@TrangPham If you like this organization better, I would only ask for a couple of aesthetic changes:
1) Put the new private methods below the `_build_column_value_missing_rule()` method.
2) Set up type hints for the values returned by these new private methods -- where they are called.

However, if I had to do a refactor, I would do it at the "seem" boundary of what is needed by each `ExpectationConfigurationBuilder` level.  In other words, because we have two Expectations -- each based on its own metric -- we could create one generic method that emits the `ExpectationConfigurationBuilder` based on the input metric name as a parameter.  Bunching up ParameterBuilders does not seem very appealing to me, because they all fulfill a separate purpose, and I like seeing them with their corresponding comment in one place.  However, if you prefer, we can solve this by keeping it the way you have it and perhaps adding clarifying comment(s).